content/client-lib-development-guide/features.textile

```diff
@@ -1730,11 +1731,14 @@ class RealtimeChannel:
   publish(name: String?, data: Data?) => io // RTL6i
   subscribe((Message) ->) => io // RTL7a
   subscribe(String, (Message) ->) => io // RTL7b
+  subscribe((Message) -> Bool, (Message) ->) // RTL7g
```

Comment:
This actually wouldn't work in ably-js because `RealtimeChannelCallbacks.subscribe` is async and optionally takes a callback which resolves when the channel is attached (see https://github.com/ably/ably-js/blob/main/ably.d.ts#L715) so this signature is already being used for something else.

Also would we also want a `subscribe(String, (Message) -> Bool, (Message) ->)` signature?